src/containerapp/HISTORY.rst

```diff
@@ -7,6 +7,7 @@ upcoming
 0.3.41
 ++++++
 * 'az containerapp up/create': enable support for no Dockerfile cases with --repo
+* 'az containerapp update': fix bug for mounting secret volumes using --secret-volume-mount
```

Comment:
please add description in upcoming section.

Reply:
Done!